components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -16,6 +16,22 @@
 
 #import "MaterialShadowElevations.h"
 
+@class MDCDialogPresentationController;
+
+/**
+ MDCDialogPresentationControllerDelegate provides a method that allows a delegate of an
+ MDCDialogPresentationController to respond to its dismissals.
+ */
+@protocol MDCDialogPresentationControllerDelegate <NSObject>
+
+/**
+ This method allows a delegate conforming to MDCDialogPresentationControllerDelegate to respond to
+ MDCDialogPresentationController dismissals.
+ */
+- (void)dialogPresentationControllerDidDismiss:
+    (MDCDialogPresentationController *_Nonnull)dialogPresentationController;
```

Comment:
I believe we're able to use `nonnull` notation for nullability:

```suggestion
    (nonnull MDCDialogPresentationController *)dialogPresentationController;
```